sdks/python/apache_beam/testing/benchmarks/nexmark/models/field_name.py

```diff
@@ -0,0 +1,40 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+""" Field names for de-serializing json representation of Models
+"""
+
+
+class FieldName:
+  id = 'id'
```

Comment:
constants normally uses all capitals, like ID, EMAIL_ADDRESS, ITEM_NAME etc.

Reply:
I see I see, I can change that definitely

Reply:
I have changed them